src/components/existing-request/existing-request-display.vue

```diff
@@ -58,7 +58,10 @@
                             </v-icon>
                           </span>
                         </template>
-                        <div v-html="refundParams.refundMessage"></div>
+                        <div v-html="refundParams.refundMessageText1" />
+                        <br/>
```

Comment:
Should this line break depend on whether there is a text2?

Reply:
No, there should always be an empty line before contact-info.

Reply:
I don't mean the BR on line 66. I mean the one on line 62.

Reply:
ok. It didn't look bad with both br elements but you're right.

Reply:
Thanks! Done.